.travis.yml

```diff
@@ -1,17 +1,19 @@
 language: d
-dist: trusty
 sudo: true
+dist: bionic
 
 matrix:
   include:
     - d: dmd-nightly
-      script: dub run --root mir-integration && dub run --build=unittest -c integration-mir
-    - d: dmd-2.087.0
-    - d: dmd-2.086.1
-    - d: dmd-2.085.1
-    - d: ldc-1.16.0
-    - d: ldc-1.15.0
-    - d: ldc-1.14.0
+    - d: dmd
```

Comment:
Adding `dmd` and `ldc` without versions and not allowing failures for them makes the build not be deterministic.

Reply:
Added them to allowed failures